app/services/albums.js

```diff
@@ -0,0 +1,11 @@
+const request = require('request'),
+  errors = require('../errors');
+
+exports.getAll = () => {
+  return new Promise((resolve, reject) => {
+    request.get('https://jsonplaceholder.typicode.com/albums', (error, response, body) => {
+      if (error) reject(error);
+      resolve(JSON.parse(body));
```

Comment:
you should do this with the request options specifying that the response is a json, rather than doing a manual parse